src/NUnitEngine/nunit.engine/Properties/AssemblyInfo.cs

```diff
@@ -27,4 +27,10 @@
     "fa6d1ea760e1ca6065cee41a1a279ca234933fe977a096222c0e14f9e5a17d5689305c6d7f1206"+
     "a85a53c48ca010080799d6eeef61c98abd18767827dc05daea6b6fbd2e868410d9bee5e972a004"+
     "ddd692dec8fa404ba4591e847a8cf35de21c2d3723bc8d775a66b594adeb967537729fe2a446b5"+
-    "48cd57a6")]
\ No newline at end of file
+    "48cd57a6")]
+
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2,PublicKey=002400000480000094" +
```

Comment:
Did I miss where this is explained? Maybe a comment here would help.

Reply:
Sorry - this is to allow using NSubstitute with internal types. It should have a comment, yes!

Reply:
Done!